python/folderify/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        lgarron folderify 1.2.3 v
+github.tarball_from archive
+
+categories          python amusements
+platforms           darwin
+license             MIT
+
+maintainers         {@harens gmail.com:harensdeveloper} \
+                    openmaintainer
+description         Generate pretty, beveled macOS folder icons
+long_description    Generate a native macOS folder icon from a mask file
+
+checksums           rmd160 d8360360574827ad2852116d6e0127e708f56c12 \
+                    sha256 3a9eaadf1f2a9dde3ab58bb07ea5b1a5f5a182f62fe19e2cd79a88f6abe00f7e \
+                    size   4382181
+
+python.versions     38
+depends_lib-append  port:ImageMagick
+
+test {
+    system -W ${worksrcpath} "folderify"
```

Comment:
This test phase doesn't work:

```
sh: folderify: command not found
```

If it worked for you, it was because you already had the port installed, and it ran that instead, so that wouldn't be a valid test of what was just built.

Also, there's no reason to override the test phase and call `system` here. You can set the test phase variables normally:

```
test.run            yes
test.cmd            path/to/folderify
```

However I don't know what the correct `path/to/folderify` is; the build does not appear to produce a file called `folderify`. One does get created in the destroot phase, but that's too late to be used by the test phase.